Makefile.gen

```diff
@@ -24,10 +24,35 @@ symlink-gopath:
 	@[[ $(shell pwd) == $(shell realpath $(CANONICAL_GOPATH)) ]] \
 		|| (echo "expected $(shell pwd) to equal $(shell realpath $(CANONICAL_GOPATH))" ; exit 1)
 
+CLIENTGEN_TOOLS := client-gen deepcopy-gen informer-gen lister-gen conversion-gen
+# This should match the k8s.io/code-generator version in go.mod.
+# We do not install k8s.io/code-generator, so use client-go as a proxy.
+CLIENTGEN_VERSION := $(shell cat go.mod | grep k8s.io/client-go | head -n1 | cut -d' ' -f2)
```

Comment:
Suggest adding the before & after versions to the commit message. Looks like v0.26.9 is the client we're using now, at least until the next bump. And the old hard coded version was v0.22.2?

Reply:
Yeah, the diff will be clearer once this change is rebased - but I'll update the commit message as well

Reply:
done